jsonschema/tests/test_cli.py

```diff
@@ -683,6 +684,73 @@ def test_successful_validation_of_just_the_schema_pretty_output(self):
             stderr="",
         )
 
+    def test_successful_validate_with_specifying_base_uri_absolute_path(self):
+        absolute_path = os.getcwd()
+        try:
+            schema_file = tempfile.NamedTemporaryFile(
+                mode='w+',
+                prefix='schema',
+                suffix='.json',
+                dir=absolute_path,
```

Comment:
Not sure I follow why we're specifying all these options to `NamedTemporaryFile`. If you use `tempfile.NamedTemporaryFile(delete=False)` it should just give you an (absolute) path to a temporary file.